core/templates/dev/head/dashboard/notifications_dashboard.html

```diff
@@ -26,46 +26,53 @@ <h2 class="oppia-page-heading">Notifications</h2>
       Last updated: <[getLocaleAbbreviatedDatetimeString(jobQueuedMsec)]>
     </div>
 
-    <md-card class="oppia-page-card oppia-notifications-dashboard-card">
-      <div ng-if="recentNotifications.length === 0">
+    <div class="oppia-notifications-tray">
+      <md-card ng-if="recentNotifications.length === 0" class="oppia-page-card oppia-notifications-dashboard-card">
         <em>No recent notifications.</em>
-      </div>
+      </md-card>
+
+      <div ng-if="recentNotifications.length > 0" >
+        <md-card ng-repeat="notification in recentNotifications"
+            ng-class="{'oppia-dashboard-row-recent': lastSeenMsec && lastSeenMsec < notification.last_updated_ms && notification.author_username !== currentUsername, 'oppia-dashboard-row': !lastSeenMsec || lastSeenMsec >= notification.last_updated_ms || notification.author_username === currentUsername}"
+            ng-click="navigateToItem(notification.activity_id, notification.type)">
+            <div ng-if="notification.type === 'feedback_thread'" class="oppia-feedback-notification">
```

Comment:
Deindent this by 2, since it's only one level more than the md-card on line 35. I think you have an extra closing div tag at the end, too.